curriculum/challenges/_meta/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/meta.json

```
@@ -0,0 +1,217 @@
+{
+  "name": "Learn Recursion by Solving the Tower of Hanoi Puzzle",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-recursion-by-solving-the-tower-of-hanoi-puzzle",
+  "order": 43,
```

```suggestion
  "order": 2,
```

This should please the tests.